.gitignore

```diff
@@ -5,7 +5,7 @@ lerna-debug.log
 .npmrc
 
 # i18n
-translations/**/*.ts
+locales/**/*.ts
```

Comment:
Why we need this change?

Reply:
It's out of scope of this task. Let's keep the changes related to Crowdin.

Reply:
Done